README.md

```diff
@@ -21,21 +21,22 @@ pip install coralogix-logger
 >This part is required only if you are not in the EU region
 >The URL can be provide as an Evironment variable.
 ```bash
-export CORALOGIX_LOG_URL = 'https://<Cluster_api_URL>:443/api/v1/logs'
+export CORALOGIX_LOG_URL = 'https://<Cluster_api_URL>/api/v1/logs'
 ```
 Or from the app itself:
 ```python
 from coralogix.constants import Coralogix
-Coralogix.CORALOGIX_LOG_URL = 'https://<Cluster_api_URL>:443/api/v1/logs'
+Coralogix.CORALOGIX_LOG_URL = 'https://<Cluster_api_URL>/api/v1/logs'
 ```
 Note: Remember to set the CORALOGIX_LOG_URL before starting the CoralogixLogger.
 
 The options:
 ```
-api.coralogix.us 
-api.coralogix.com (default)
-api.app.coralogix.in
-api.coralogixsg.com
+ingress.coralogix.us 
+ingress.coralogix.com (default)
+ingress.app.coralogix.in
+ingress.coralogixsg.com
+ingress.cx498.coralogix.com
 ```
```

Comment:
I forgot to add ingress.eu2.coralogix.com which is used in the app.py and I tested